.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,53 @@
+---
+name: Bug report
+about: Unexpected behavior? Tell us more
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Self checklist**
+
+ - [ ] I checked that this issue hasn't been created before
+ - [ ] I added any relevant label to the issue
```

Comment:
How should a person, external to inmanta know what the relevant label is?

Reply:
Checklist has been removed in https://github.com/inmanta/inmanta/pull/2416/commits/35fb7151fd7c0a968dadf87696f42179d272e650